src/renderer/features/extension-wallet/index.tsx

```diff
@@ -31,7 +31,14 @@ extensionWalletFeature.inject(accountService.accountActionPermissionAnyOf, ({ ac
 extensionWalletFeature.inject(walletIconSlot, ({ wallet, size }) => {
   if (!polkadotExtensionService.isExtensionWallet(wallet)) return null;
 
-  return <Icon name={walletIcon[wallet.type].icon} size={size} />;
+  return (
+    <div className="relative">
+      <Identicon address={wallet.accounts[0].accountId} size={size} background={false} />
```

Comment:
Same here

Reply:
Maybe you should move this Identicon + wallet icon component to `ui-entities`? Name it `WalletAccountIcon` idk